frontend/public/components/masthead-toolbar.jsx

```diff
@@ -52,6 +63,9 @@ class MastheadToolbar_ extends React.Component {
   }
 
   componentDidMount() {
+    if (window.SERVER_FLAGS.statuspageID) {
+      this._getStatuspageData(window.SERVER_FLAGS.statuspageID);
```

Comment:
It would be nice if we could push this logic down into the `SystemStatusButton` button component, but since it's on the page twice, it would mean two requests.

Another option would be to put it in Redux. Then we could surface incidents in other places like the cluster dashboard.

I think this is probably OK, though.